calicoctl/calicoctl.go

```diff
@@ -54,7 +54,15 @@ Description:
 
   See 'calicoctl <command> --help' to read about a specific subcommand.
 `
-	arguments, _ := docopt.Parse(doc, nil, true, commands.VERSION_SUMMARY, true, false)
+	arguments, err := docopt.Parse(doc, nil, true, commands.VERSION_SUMMARY, true, false)
+
```

Comment:
just a minor formatting thing, but generally would remove this newline between the call which generates the error, and the handling of the error.

e.g.,

```go
retVal, err := someFunc()
if err !=nil {
  // Handle it
  ...
}
```

Reply:
Ah, ok.

Let me patch that up tomorrow morning and send another commit.

Reply:
I added that whitespace commit, let me know.

Reply:
Looks nice